functions/go/starlark/starlark/processor.go

```diff
@@ -0,0 +1,37 @@
+package starlark
+
+import (
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+)
+
+type Processor struct{}
+
+func Process(resourceList *fn.ResourceList) (bool, error) {
+	err := func() error {
+		sfc := &StarlarkFnConfig{}
+		if err := sfc.Decode(resourceList.FunctionConfig); err != nil {
+			return err
+		}
+		if err := sfc.Default(); err != nil {
+			return err
+		}
+		if err := sfc.Validate(); err != nil {
+			return err
+		}
+		if te := sfc.Evaluate(resourceList); te != nil {
+			return te
+		}
+		return nil
```

Comment:
Suggest adding docstring to the public functions since they are not following the typical `Config` `Transform` convention.

Actually, do you think we can merge Decode, Default and Validate to `Config`? Functions here are served as example code for SDK users, so they should follow the same pattern.